core/src/main/java/com/graphhopper/routing/util/parsers/AbstractAverageSpeedParser.java

```diff
@@ -38,9 +38,7 @@ public double getMaxSpeed() {
     public static double getMaxSpeed(ReaderWay way, boolean bwd) {
         double maxSpeed = OSMValueExtractor.stringToKmh(way.getTag("maxspeed"));
         double directedMaxSpeed = OSMValueExtractor.stringToKmh(way.getTag(bwd ? "maxspeed:backward" : "maxspeed:forward"));
-        if (isValidSpeed(directedMaxSpeed) && (!isValidSpeed(maxSpeed) || directedMaxSpeed < maxSpeed))
-            maxSpeed = directedMaxSpeed;
-        return maxSpeed;
```

Comment:
Do you think so? For example I would interpret `maxspeed=100;maxspeed:forward=50` such that 100 is meant for both directions but the forward direction gets overwritten. But I could be wrong. Either way it should be consistent with what we do in OSMMaxSpeedParser

Reply:
Yes this would have no advantage over tagging forward+backward instead of using maxspeed, or maybe when different transportation modes would be involved? But either way since it is possible people will do it.